### DISCUSSION TITLE
Copying Resources to Colleagues

### DISCUSSION THREAD

**Discussion by BeardBrian** [EsriPS/arcgis-assistant-feedback#58] [Q&A]
I am a high school teacher who uses AGOL and wants to help others. I am wanting to create Survey123, web maps and other items and copy them to other users who are less comfortable.

I have admin privileges in our organization. 

When I go to copy an item to a colleague I get "CONT_0004: User folder does not exist." even though I see the folder. Other items it has said "Coped 1 item" but we do not see the content in their Content.

**Comment by jpeterson** [comment]
Hi @BeardBrian, thanks for reaching out! Here are a few follow-up questions so we can pinpoint what may be going on here:

1. Can you share the exact steps you're following in order to copy the item?
2. When copying, are you creating a new folder or copying into an existing folder?
3. Is the user you are copying the item to able to own content (e.g., they are not a Viewer)?
4. Can you attempt the following:
	a. Use Assistant to copy the item into your own account instead of your colleague's
	b. Use ArcGIS Online's [Change owner feature](https://doc.arcgis.com/en/arcgis-online/administer/manage-items.htm#ESRI_SECTION1_F9D72CA4252E4EB18F4B081C27ADF527) to change ownership of the copied item to your colleague
	c. Was this successful? If not, it may illuminate something in the way your organization/users are configured. If it did work, I'd suggest this workflow as a temporary workaround until we can get this sorted in Assistant.

Thanks in advance for sharing your feedback!

**Reply by BeardBrian** [reply]
Sorry for the delay! I was able to copy items from one of my accounts into my new one which was great. I will test with a colleague again soon and get back to you; however, I noticed that I could not copy over Story Maps. Is that not possible?
